accumulator.h

```diff
@@ -0,0 +1,23 @@
+#include <string>
+class base
```

Comment:
Only one class derives from `base`. Why isn't it together with `BMS_Car_System`?